pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -330,3 +340,59 @@ func collectVMIInterfaceInfo(vmi *k6tv1.VirtualMachineInstance, iface k6tv1.Virt
 		Value: 1.0,
 	}
 }
+
+func CollectVmisVnicInfo(vmi *k6tv1.VirtualMachineInstance) []operatormetrics.CollectorResult {
+	var results []operatormetrics.CollectorResult
+
+	interfaces := vmi.Spec.Domain.Devices.Interfaces
+	networks := vmi.Spec.Networks
+
+	for _, iface := range interfaces {
+		bindingType := none
```

Comment:
@avlitman perhaps better to add a const for "none".

Reply:
now this code moved to a separate functions, so added `const none = "none"`  for both getBinding, getNetworkName funcs

Reply:
discussed in other comment: https://github.com/kubevirt/kubevirt/pull/13610#discussion_r1900384329
closing this one